libs/remix-lib/src/execution/txRunnerWeb3.ts

```diff
@@ -57,6 +60,10 @@ export class TxRunnerWeb3 {
         return new Promise(async (resolve, reject) => {
           const receipt = await tryTillReceiptAvailable(resp, this.getWeb3())
           tx = await tryTillTxAvailable(resp, this.getWeb3())
+          currentDateTime = new Date();
+          const end = currentDateTime.getTime() / 1000
+          console.log('end', end)
+          console.log('stopwatch', end - start)
```

Comment:
would you like to keep these two logs too?

Reply:
??

Reply:
I removed the other logs and kept this one.